firmware/application/gradient.cpp

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (C) 2025 Belousov Oleg
+ *
+ * This file is part of PortaPack.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2, or (at your option)
+ * any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; see the file COPYING.  If not, write to
+ * the Free Software Foundation, Inc., 51 Franklin Street,
+ * Boston, MA 02110-1301, USA.
+ */
+
+#include "gradient.hpp"
+
+#include "convert.hpp"
+#include "file_reader.hpp"
+namespace fs = std::filesystem;
+
+const std::filesystem::path default_gradient_file = u"waterfall.txt";
+
+Gradient::Gradient() {
+    prev_index = 0;
+    prev_r = 0;
+    prev_g = 0;
+    prev_b = 0;
+}
+
+void Gradient::set_default() {
```

Comment:
can this also be turned into a WATERFALLS/default.txt settings please ?

Reply:
I mean, keep the set default so in case there is not Sdcard we're still having a color scheme, but having also a default.txt file in WATERFALLS is more user friendly IMHO

Reply:
Okay I've done it :-)